qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -112,6 +112,7 @@ def test_minimal_health(self):
                     JObj({
                         'in': int,
                         'up': int,
+                        'state': JList(str)
```

Comment:
This technically means we're altering the format of the `/health` endpoint, right? Since it's additive, perhaps we should increase only the minor version, but unfortunately we don't have yet the code in place for dealing with multiple minor versions... What do you think @aaSharma14 @alfonsomthd @avanthakkar @nizamial09 ?